diff --git a/node_modules/@mgcrea/fastify-session/lib/session/Session.js b/node_modules/@mgcrea/fastify-session/lib/session/Session.js
index 391c1b3..7187816 100644
--- a/node_modules/@mgcrea/fastify-session/lib/session/Session.js
+++ b/node_modules/@mgcrea/fastify-session/lib/session/Session.js
@@ -36,7 +36,8 @@ class Session {
         const { buffer: cleartext, rotated } = Session[exports.kSessionCrypto].unsealMessage(cookie, Session[exports.kSecretKeys]);
         // Stateless sessions have the whole session data encrypted as the cookie
         if (Session[exports.kSessionCrypto].stateless) {
-            const session = new Session(JSON.parse(cleartext.toString()));
+            const data = JSON.parse(cleartext.toString())
+            const session = new Session(data, {id: data.id})
             session.rotated = rotated;
             return session;
         }
