diff --git a/node_modules/@nuxt/nitro/dist/index.cjs b/node_modules/@nuxt/nitro/dist/index.cjs
index 093d604..67a6fcf 100644
--- a/node_modules/@nuxt/nitro/dist/index.cjs
+++ b/node_modules/@nuxt/nitro/dist/index.cjs
@@ -803,7 +812,7 @@ const getRollupConfig = (nitroContext) => {
     },
     treeshake: {
       moduleSideEffects(id) {
-        return nitroContext.moduleSideEffects.some((match) => id.startsWith(match));
+        return id.includes("node_modules") || nitroContext.moduleSideEffects.some((match) => id.startsWith(match));
       }
     }
   };
diff --git a/node_modules/@nuxt/nitro/dist/runtime/app/render.mjs b/node_modules/@nuxt/nitro/dist/runtime/app/render.mjs
index 03bd228..b60a9ba 100644
--- a/node_modules/@nuxt/nitro/dist/runtime/app/render.mjs
+++ b/node_modules/@nuxt/nitro/dist/runtime/app/render.mjs
@@ -27,7 +27,7 @@ const getSPARenderer = cachedResult(async () => {
       html: '<div id="__nuxt"></div>',
       renderResourceHints: () => "",
       renderStyles: () => "",
-      renderScripts: () => clientManifest.initial.map((s) => {
+      renderScripts: () => (clientManifest.initial || []).map((s) => {
         const isMJS = !s.endsWith(".js");
         return `<script ${isMJS ? 'type="module"' : ""} src="${clientManifest.publicPath}${s}"><\/script>`;
       }).join("")
