diff --git a/node_modules/@nuxt/kit/dist/index.cjs b/node_modules/@nuxt/kit/dist/index.cjs
index 772951a..cc2abde 100644
--- a/node_modules/@nuxt/kit/dist/index.cjs
+++ b/node_modules/@nuxt/kit/dist/index.cjs
@@ -104,12 +104,23 @@ function tryResolveModule(path, opts = {}) {
     }
   }
 }
-function requireModule(id, opts = {}) {
+function requireModule(id, opts = {}, jopts) {
   const resolvedPath = resolveModule(id, opts);
   if (opts.clearCache && !isNodeModules(id)) {
     clearRequireCache(resolvedPath);
   }
-  let requiredModule = _require(resolvedPath);
+  let requiredModule = jiti__default(process.cwd(), !jopts ? {} : {
+		transformOptions: {
+			babel: {
+				plugins: [
+					[require('babel-plugin-module-resolver'), {
+						root: [jopts.srcDir],
+						alias: jopts.alias,
+					}],
+				],
+			},
+		},
+	})(resolvedPath);
   if (opts.interopDefault !== false && requiredModule && requiredModule.default) {
     requiredModule = requiredModule.default;
   }
@@ -1145,7 +1156,7 @@ async function installModule(nuxt, installOpts) {
   let handler;
   if (typeof src === "string") {
     const _src = resolveModule(resolveAlias(src, nuxt.options.alias), { paths: nuxt.options.modulesDir });
-    handler = requireModule(_src);
+    handler = requireModule(_src, {}, nuxt.options);
     if (!meta.name) {
       meta.name = src;
     }
