diff --git a/node_modules/@nuxt/nitro/dist/index.mjs b/node_modules/@nuxt/nitro/dist/index.mjs
index 3ae7dd0..3148b47 100644
--- a/node_modules/@nuxt/nitro/dist/index.mjs
+++ b/node_modules/@nuxt/nitro/dist/index.mjs
@@ -776,7 +776,7 @@ const getRollupConfig = (nitroContext) => {
     },
     treeshake: {
       moduleSideEffects(id) {
-        return nitroContext.moduleSideEffects.some((match) => id.startsWith(match));
+        return id.includes("node_modules") || nitroContext.moduleSideEffects.some((match) => id.startsWith(match));
       }
     }
   };
@@ -854,6 +854,7 @@ const getRollupConfig = (nitroContext) => {
       "@/": nitroContext._nuxt.srcDir,
       "~~": nitroContext._nuxt.rootDir,
       "@@/": nitroContext._nuxt.rootDir,
+      ...nitroContext.__alias,
       ...env.alias
     }
   }));
@@ -1568,6 +1569,7 @@ function getNitroContext(nuxtOptions, input) {
       inline: !nuxtOptions.dev,
       dirs: {}
     },
+     __alias: nuxtOptions.alias || {},
     _nuxt: {
       majorVersion: nuxtOptions._majorVersion || 2,
       dev: nuxtOptions.dev,
