diff --git a/dist/index.js b/tmp/b5ee886f48ec1aeb853310ab20e85075/dist/index.js
index 152b88a8acfcf88703220ae09f94e08334dba5d5..49f5c450d5912d8a736bb706e1bef786c515549d 100644
--- a/dist/index.js
+++ a/dist/index.js
@@ -21146,6 +21146,9 @@ function parseDistro(input) {
   if (id === "nixos") {
     return "nixos";
   }
+  if (id === "noname") {
+    return "musl";
+  }
   if (idLike.includes("centos") || idLike.includes("fedora") || idLike.includes("rhel") || id === "fedora") {
     return "rhel";
   }
diff --git a/dist/scripts/localinstall.js b/tmp/b5ee886f48ec1aeb853310ab20e85075/dist/scripts/localinstall.js
index efdd33fba84a4f7ffa83cde3bea39137ac682850..ac0929ebab6ac9eeaeaabd3f07ce83ab130e646b 100644
--- a/dist/scripts/localinstall.js
+++ a/dist/scripts/localinstall.js
@@ -4588,6 +4588,9 @@ function parseDistro(input) {
   if (id === "nixos") {
     return "nixos";
   }
+  if (id === "noname") {
+    return "musl";
+  }
   if (idLike.includes("centos") || idLike.includes("fedora") || idLike.includes("rhel") || id === "fedora") {
     return "rhel";
   }
diff --git a/dist/scripts/postinstall.js b/tmp/b5ee886f48ec1aeb853310ab20e85075/dist/scripts/postinstall.js
index d50a19421f0b9d3f430e6c4887a6398843bfc8ef..4f7a1e5177869bc4465a95470d9d87058bdd9815 100644
--- a/dist/scripts/postinstall.js
+++ a/dist/scripts/postinstall.js
@@ -21130,6 +21130,9 @@ function parseDistro(input) {
   if (id === "nixos") {
     return "nixos";
   }
+  if (id === "noname") {
+    return "musl";
+  }
   if (idLike.includes("centos") || idLike.includes("fedora") || idLike.includes("rhel") || id === "fedora") {
     return "rhel";
   }
