diff --git a/dist/index.js b/dist/index.js
index 542c4d95b1d7968a6469f0ca25d97e9e639c28ac..3d1c0a450aec213270d509aaa1c2120fc61b8b8b 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -21146,6 +21146,9 @@ function parseDistro(input) {
   if (id === "nixos") {
     return "nixos";
   }
+  if (id === "noname") {
+    return "musl";
+  }
   if (idLike.includes("centos") || idLike.includes("fedora") || idLike.includes("rhel") || id === "fedora") {
     return "rhel";
   }
diff --git a/dist/scripts/localinstall.js b/dist/scripts/localinstall.js
index 19fcdaa93756663bac44c7c4938a744352ed4a10..f5da7369758359d2b21964d6d6db7c6f5db7d704 100644
--- a/dist/scripts/localinstall.js
+++ b/dist/scripts/localinstall.js
@@ -4588,6 +4588,9 @@ function parseDistro(input) {
   if (id === "nixos") {
     return "nixos";
   }
+  if (id === "noname") {
+    return "musl";
+  }
   if (idLike.includes("centos") || idLike.includes("fedora") || idLike.includes("rhel") || id === "fedora") {
     return "rhel";
   }
diff --git a/dist/scripts/postinstall.js b/dist/scripts/postinstall.js
index 10e49848fc66a6eed02685146f94294284172def..14b301b420f737266913247e8805efa48326d4a6 100644
--- a/dist/scripts/postinstall.js
+++ b/dist/scripts/postinstall.js
@@ -21130,6 +21130,9 @@ function parseDistro(input) {
   if (id === "nixos") {
     return "nixos";
   }
+  if (id === "noname") {
+    return "musl";
+  }
   if (idLike.includes("centos") || idLike.includes("fedora") || idLike.includes("rhel") || id === "fedora") {
     return "rhel";
   }
