--- a/dist/shared/preset-mini.3602602e.mjs
+++ b/dist/shared/preset-mini.3602602e.mjs
@@ -485,7 +485,7 @@ const contentVisibility = [
   ...makeGlobalStaticRules("content-visibility")
 ];
 const contents = [
-  [/^content-\[(.+)\]$/, ([, v]) => ({ content: `"${v}"` })],
+  [/^content-\[(.+)\]$/, ([, v]) => ({ content: /^attr\(.+\)$/.test(v) ? `${v}` : `"${v}"` })],
   [/^content-(\$.+)]$/, ([, v]) => ({ content: handler.cssvar(v) })],
   ["content-empty", { content: '""' }],
   ["content-none", { content: '""' }]
--- a/dist/shared/preset-mini.b300afab.cjs
+++ b/dist/shared/preset-mini.b300afab.cjs
@@ -487,7 +487,7 @@ const contentVisibility = [
   ...colors.makeGlobalStaticRules("content-visibility")
 ];
 const contents = [
-  [/^content-\[(.+)\]$/, ([, v]) => ({ content: `"${v}"` })],
+  [/^content-\[(.+)\]$/, ([, v]) => ({ content: /^attr\(.+\)$/.test(v) ? `${v}` : `"${v}"` })],
   [/^content-(\$.+)]$/, ([, v]) => ({ content: colors.handler.cssvar(v) })],
   ["content-empty", { content: '""' }],
   ["content-none", { content: '""' }]
