diff --git a/dist/shared/preset-mini.3602602e.mjs b/dist/shared/preset-mini.3602602e.mjs
index 5b1aea5b1bf72157e10f56ea2f0830bec9b75294..f404f1312304acdf236f55e29f8e85994c57a936 100644
--- a/dist/shared/preset-mini.3602602e.mjs
+++ b/dist/shared/preset-mini.3602602e.mjs
@@ -486,6 +486,7 @@ const contentVisibility = [
 ];
 const contents = [
   [/^content-\[(.+)\]$/, ([, v]) => ({ content: `"${v}"` })],
+  [/^content-raw-\[(.+)\]$/, ([, v]) => ({ content: `${v}` })],
   [/^content-(\$.+)]$/, ([, v]) => ({ content: handler.cssvar(v) })],
   ["content-empty", { content: '""' }],
   ["content-none", { content: '""' }]
diff --git a/dist/shared/preset-mini.b300afab.cjs b/dist/shared/preset-mini.b300afab.cjs
index a69946c7d5b9dff309a54ba20ec0ee5bf97d3ecc..82d89f8afc3aaa5406a142f463f9e2133684a60d 100644
--- a/dist/shared/preset-mini.b300afab.cjs
+++ b/dist/shared/preset-mini.b300afab.cjs
@@ -488,6 +488,7 @@ const contentVisibility = [
 ];
 const contents = [
   [/^content-\[(.+)\]$/, ([, v]) => ({ content: `"${v}"` })],
+  [/^content-raw-\[(.+)\]$/, ([, v]) => ({ content: `${v}` })],
   [/^content-(\$.+)]$/, ([, v]) => ({ content: colors.handler.cssvar(v) })],
   ["content-empty", { content: '""' }],
   ["content-none", { content: '""' }]