diff --git a/lib/main.js b/tmp/a647fee82cf8f8f2086675197d905fcf/lib/main.js
index 193fddebe429ea421ad328c208e31e0be07e5866..3f5dee447ca3816220e3f77e75da5083a5f9b338 100644
--- a/lib/main.js
+++ b/lib/main.js
@@ -1260,6 +1260,11 @@ function createChannel(streamIn) {
       copyResponseToResult(response, result);
       runOnEndCallbacks(result, logPluginError, () => {
         if (result.errors.length > 0) {
+					if (response.rebuild) {
+            sendRequest(refs, { command: 'rebuild-dispose', key }, () => {
+              // We don't care about the result
+            });
+          }
           return callback2(failureErrorWithLog("Build failed", result.errors, result.warnings), null);
         }
         if (response.rebuild) {
