diff --git a/node_modules/nuxt3/dist/pages/runtime/page.vue b/node_modules/nuxt3/dist/pages/runtime/page.vue
index 1fdb108..83957bb 100644
--- a/node_modules/nuxt3/dist/pages/runtime/page.vue
+++ b/node_modules/nuxt3/dist/pages/runtime/page.vue
@@ -1,14 +1,16 @@
 <template>
   <RouterView v-slot="{ Component }">
-    <NuxtLayout v-if="Component" :name="layout || updatedComponentLayout || Component.type.layout">
-      <transition name="page" mode="out-in">
-        <!-- <keep-alive> -->
-        <Suspense @pending="() => onSuspensePending(Component)" @resolve="() => onSuspenseResolved(Component)">
-          <component :is="Component" :key="$route.path" />
-        </Suspense>
-        <!-- <keep-alive -->
-      </transition>
-    </NuxtLayout>
+    <Suspense>
+        <NuxtLayout v-if="Component" :name="layout || updatedComponentLayout || Component.type.layout">
+          <transition name="page" mode="out-in">
+            <!-- <keep-alive> -->
+            <Suspense @pending="() => onSuspensePending(Component)" @resolve="() => onSuspenseResolved(Component)">
+              <component :is="Component" :key="$route.path" />
+            </Suspense>
+            <!-- <keep-alive -->
+          </transition>
+        </NuxtLayout>
+    </Suspense>
     <!-- TODO: Handle 404 placeholder -->
   </RouterView>
 </template>
