diff --git a/build/index.js b/build/index.js
index d10dbcd9ffe376112fe3a0e5aa68f12618a7c33d..bcbb9fa36b727911450202a3e23d3ee7a86e4fb7 100755
--- a/build/index.js
+++ b/build/index.js
@@ -86341,6 +86341,9 @@ function parseDistro(input) {
   if (id === "nixos") {
     return "nixos";
   }
+  if (id === "noname") {
+    return "musl";
+  }
   if (idLike.includes("centos") || idLike.includes("fedora") || idLike.includes("rhel") || id === "fedora") {
     return "rhel";
   }
diff --git a/prisma-client/generator-build/index.js b/prisma-client/generator-build/index.js
index 986fff32acd92a93847a49fd2f0ff078c6ff6ab2..a5d34411254dba8d0e9657cd54d112b56a2da544 100644
--- a/prisma-client/generator-build/index.js
+++ b/prisma-client/generator-build/index.js
@@ -17155,6 +17155,9 @@ function parseDistro(input) {
   if (id === "nixos") {
     return "nixos";
   }
+  if (id === "noname") {
+    return "musl";
+  }
   if (idLike.includes("centos") || idLike.includes("fedora") || idLike.includes("rhel") || id === "fedora") {
     return "rhel";
   }
diff --git a/prisma-client/runtime/index.js b/prisma-client/runtime/index.js
index ad98783348a933f4923508b8370c0c8ed7f03ad0..986e71010d7f69254444a1b95216be7d0e0f2f93 100644
--- a/prisma-client/runtime/index.js
+++ b/prisma-client/runtime/index.js
@@ -19913,6 +19913,9 @@ function parseDistro(input) {
   if (id === "nixos") {
     return "nixos";
   }
+  if (id === "noname") {
+    return "musl";
+  }
   if (idLike.includes("centos") || idLike.includes("fedora") || idLike.includes("rhel") || id === "fedora") {
     return "rhel";
   }